### PR TITLE
Helper functor in PolygonalCalculus to project vertices onto their tangent plane.

### PR DESCRIPTION
**Pull request by dcoeurjo**
# PR Description

aka.  Corrected calculus from projection.

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug mode.
- [x] All continuous integration tests pass (Github Actions)

### CODE REVIEW

**Comment by dcoeurjo** [comment]
@kerautret @JacquesOlivierLachaud could you please have a Quick Look to this one?

**Comment by dcoeurjo** [comment]
The Conan issue has been fixed in the rel14. PR

File: src/DGtal/dec/doc/modulePolygonalCalculus.dox [hunk 1]
```diff
@@ -253,40 +257,75 @@ used when one solves a weak problem and wishes to get a pointwise
...
+ one can implicitly project each face to a prescribed tangent plane and perform the computations on this  new embedding of the geometry.
 
-\subsection Global Vector Calculus
+Geodesic distances without correction | Geodesic distances with correction
+ --|--
+@image html images/poly/corrected-without.png "" | @image html images/poly/corrected-with.png ""
 
-Global Vector Laplace/Poisson problems can also be solved by the same way, using instead PolygonalCalculus<SurfMesh>::globalConnectionLaplace() and PolygonalCalculus<SurfMesh>::doubledGlobalLumpedMassMatrix(). One can find examples of such use in the \ref VectorsInHeat class.
+ 
+The functor functors::EmbedderFromNormalVectors can be used to implicitly project Face vertices onto the prescribed tangent plane. A calssical usage is the following one:
```
Comment (JacquesOlivierLachaud) [inline src/DGtal/dec/doc/modulePolygonalCalculus.dox:273]: A classical usage

Comment (dcoeurjo) [inline src/DGtal/dec/doc/modulePolygonalCalculus.dox:273]:
```suggestion
The functor functors::EmbedderFromNormalVectors can be used to implicitly project Face vertices onto the prescribed tangent plane. A classical usage is the following one:
```

Comment (dcoeurjo) [reply src/DGtal/dec/doc/modulePolygonalCalculus.dox]: thx done

**Pull request merged**